src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -818,9 +821,9 @@ public override BoundNode VisitLocal(BoundLocal node)
             LocalSymbol localSymbol = node.LocalSymbol;
 
             Debug.Assert(!IsConditionalState);
-            if (this.State.Reachable)
+            //if (this.State.Reachable)
```

Comment:
>//if (this.State.Reachable) [](start = 12, length = 27)

Commented out code without PROTOTYPE comment.

Reply:
Or should this code be un-commented?

---
In reply to: [155057239](https://github.com/dotnet/roslyn/pull/23439#discussion_r155057239) [](ancestors = 155057239)